components/eamxx/src/share/util/scream_universal_constants.hpp

```diff
@@ -1,13 +1,18 @@
 #ifndef SCREAM_UNIVERSAL_CONSTANTS_HPP
 #define SCREAM_UNIVERSAL_CONSTANTS_HPP
 
+#include <limits>
+
 namespace scream {
 
 namespace constants {
 
 constexpr int seconds_per_day       = 86400;
 constexpr int days_per_nonleap_year = 365;
 
+// Universal fill value for variables
+constexpr float DEFAULT_FILL_VALUE = std::numeric_limits<float>::max() / 1e5;
```

Comment:
Why is this hard-coded to be a float? I would prefer something templated on type. E.g.
```
template<typename T>
struct DefaultFillValue {
   constexpr T value = std::numeric_limits<float>::max();
};
```
You would have to use it as `DefaultFillValue<float>::value`, but that's at least more flexible.

Reply:
Btw, the syntax burden is just temporary, since when we switch to C++17 we'll be able to do this much more concisely:

```
template<typename T>
inline constexpr T defaultFillValue = std::numeric_limits<T>::max();
```
which can then be used as `defaultFillValue<float>`.

Reply:
Yeah, I get the use case, but since this is in the universal constants file rather than some file in the io folder, I think we should be more general.